cypress/integration/iframe.spec.ts

```diff
@@ -0,0 +1,24 @@
+import {IframePage} from "../page";
+
+let iframePage: IframePage;
+
+describe("Working with Iframes", () => {
+  before(() => {
+    iframePage = new IframePage();
+  });
+  describe("When the user go the Iframe page", () => {
+    it("Should have the Tittle HTML Tutorial", () => {
+      // Arrange
+      const HTMLTutorialExpected = "HTML Tutorial";
+      const CSSTutorialExpected = "CSS Tutorial";
+
+      // Act
+      iframePage.visit();
+
+      // Assert
+      iframePage.getFrameTitle().should("contain.text", HTMLTutorialExpected);
+      iframePage.goToCssPageInFrame();
+      iframePage.getFrameTitle().should("contain.text", CSSTutorialExpected);
+    });
```

Comment:
- are expected two separated tests, you can create a before hook and put the call to the visit method there (which is not the act, it is part of the arrange)
- first test, should have the call to the getFrameTitle with the assertion of the HTML tutorial.
- The second test should have the call to the goToCssPageInFrame and the call again to the getFrameTitle with the assertion of the CSS tutorial.